cmds/nm/nm.go

```diff
@@ -0,0 +1,39 @@
+// The nm command prints out information about binaries.
+// At this point, there is very little flexibility in what it
+// does and it only does PE files, but more will come.
+package main
+
+import (
+	"debug/pe"
+	"fmt"
+	"os"
+)
+
+func syms(n string) error {
+	f, err := pe.Open(n)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%v: FileHeader %v *OptionalHeader %v\n", n, f.FileHeader, f.OptionalHeader)
+	fmt.Printf("%d Symbols: %v:\n", len(f.Symbols), f.Symbols)
```

Comment:
Doesn't this print already print all symbols? why do we print them individually again?